dcompiler.dd

```diff
@@ -85,22 +85,53 @@ C:\&gt;unzip dmc.zip
 
     $(INSTALLATION_SCRIPT_HINT)
 
-    It will create
-    a $(D ~/dmd) directory with all the files in it.
+    Unzip the archive into your home directory.
+    It will create a $(D ~/dmd) directory with all the files in it.
 
     All the tools are command line tools, which means
-    they are run from a console window.)
-    $(LI All tools can be run directly from the archive:
+    they are run from a console window.
+    All tools can be run directly from the archive:
 $(CONSOLE
 ~/dmd/linux/bin64/dmd
 )
-    To install $(D dmd) globally, add this bin folder to your $(B PATH).
+    To install $(D dmd) globally, add the `~/dmd/linux/bin64` folder to your $(B PATH).
+$(CONSOLE
+export PATH="$HOME/dmd/linux/bin64:$PATH"
+)
+
+    Alternatively, you can install dmd locally yourself by:
+    $(P $(UL
+
+    $(LI Copy $(D $(DMD_CONF)) to $(D /etc):
+
+$(CONSOLE
+sudo cp $(DMDDIR)/linux/bin64/dmd.conf /etc/$(DMD_CONF)
+)
+    )
+
+    $(LI Copy binaries to $(D /usr/local/bin):
+
+$(CONSOLE
+sudo cp $(DMDDIR)/linux/bin64/{dmd,$(DUMPOBJ),$(OBJ2ASM),$(SHELL),rdmd} /usr/local/bin
+)
+    )
+
+    $(LI Copy the library to $(D /usr/local/lib):
```

Comment:
The user will also need to copy the "headers":
* `dmd2/src/druntime/import/`
* `dmd2/src/phobos/std/`
* `dmd2/src/phobos/etc/`